internal/run/run.go

```diff
@@ -265,7 +275,7 @@ func (w *Workflow) runTarget(ctx context.Context, target string) error {
 	ctx = log.With(ctx, logger)
 	go genStep.ListenForSubsteps(logListener)
 
-	if err := sdkgen.Generate(
+	hasGenerationAccess, err := sdkgen.Generate(
```

Comment:
This seems like a pretty weird place to do this check to be honest? I would expect that we would call `sdkgen.HasGenerationAcess()` or whatever _before_ calling `generate`. I guess it doesn't really matter ultimately but still